### DISCUSSION TITLE
API inaccessible on without docker instance

### DISCUSSION THREAD

**Discussion by ani-6** [Kovah/LinkAce#837] [Support]
I have linkace v1.15.2 installed as subdir setup on my ubuntu server with apache2. App is working fine but when I try to call any api it always returns login page in response.
Also checked server error logs and application logs both are empty.

**Comment by Kovah** [comment]
Hi, thanks for trying LinkAce. Unfortunately, installation in a subdirectory is not supported yet and I have no plans for adding it at the moment.
